app/collins/softlayer/SoftLayerPlugin.scala

```diff
@@ -147,16 +149,27 @@ class SoftLayerPlugin(app: Application) extends Plugin with SoftLayer {
       .buildPut(value)
     makeRequest(request) map { r =>
       true
-    } handle {
+    } recover {
       case e => false
     }
   }
 
   protected def makeRequest(request: HttpRequest): Future[HttpResponse] = {
     val client: Service[HttpRequest,HttpResponse] = clientSpec.build()
-    client(request) ensure {
```

Comment:
If its any help, I have another PR, right behind this which changes this class to not use the finagle library, and only to use the Play WS library.  That scraps all this conversion code, as the WS library returns Scala futures natively.